src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java

```diff
@@ -60,4 +64,37 @@ public Response listAction() {
                 .ok(new ListOrderDTO(GetOrderDTO.createMany(orders)))
                 .build();
     }
+
+    @GET
+    @Path("/all")
+    @Authorized //todo make it admin-only
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Fetch list of all orders",
+            tags = { "Order" },
+            responses = {
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(schema = @Schema(implementation = ListOrderDTO.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "500",
+                            content = @Content(schema = @Schema(implementation = ExceptionDTO.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "400",
+                            content = @Content(schema = @Schema(implementation = ExceptionDTO.class))
+                    )
+            }
+    )
+    public Response listAllAction() {
```

Comment:
I think whole admin api should be placed into separate package to not confuse things together. Let's say `lt.vu.web.api.v1.admin.{same as regular api stuff}`